app/routes/start.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const router = require('express').Router();
+
+module.exports = router;
+
+
+/// View the home page
+router.get('/', async function (req, res, next) {
+  // If the user is not logged in, go to the next function below
+  if (!req.user) {
+    return next();
+  }
+
+  // If the user is logged in and they have a specified start page, redirect there
+  if (req.user.user_metadata && req.user.user_metadata.start_page) {
```

Comment:
@micchickenburger Not 100% sure.  Would it be better to do something like `if(!run.hasOwnProperty('user'))` for the previous `if` statement, so that we make sure that `req.user` is definitely defined at this line?  Similarly to one of your other comments, I think as it is right now, that previous `if` would still return `true` if `req.user` is undefined?